From ec92c3d39e0fdd0a31e85c79cda32de0a387d64a Mon Sep 17 00:00:00 2001
From: Rander Wang <rander.wang@linux.intel.com>
Date: Tue, 24 Jul 2018 11:11:14 +0800
Subject: [PATCH 1840/2345] ASoC: SOF: Remove redundant code in irq function on
 cnl

The HIPCTDR is queried two times in this function.
The first query is removed, we keep the second query
since the result is immediately used in a test

Signed-off-by: Rander Wang <rander.wang@linux.intel.com>
---
 sound/soc/sof/intel/cnl.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/sof/intel/cnl.c b/sound/soc/sof/intel/cnl.c
index 2380316..f98e699 100644
--- a/sound/soc/sof/intel/cnl.c
+++ b/sound/soc/sof/intel/cnl.c
@@ -53,7 +53,6 @@ static irqreturn_t cnl_ipc_irq_thread(int irq, void *context)
 		return ret;
 
 	hipcida = snd_sof_dsp_read(sdev, HDA_DSP_BAR, CNL_DSP_REG_HIPCIDA);
-	hipctdr = snd_sof_dsp_read(sdev, HDA_DSP_BAR, CNL_DSP_REG_HIPCTDR);
 
 	/* reply message from DSP */
 	if (hipcida & CNL_DSP_REG_HIPCIDA_DONE) {
-- 
2.7.4

